src/ReactiveDomain/ReactiveDomain/Messaging/ChainedCommand.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+using System.Threading;
+using ReactiveDomain.Messages;
+
+namespace ReactiveDomain.Messaging
+{
+    public class ChainedCommand : Command, IChainedMessage
+    {
```

Comment:
I don't see any unit tests for these Chained commands and messages.

Reply:
there does seem to be anything to test, they are just dto classes at this point